app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    args = command.split()[1:]
+    source = args[0]
+    destination = args[1]
```

Comment:
Validate command length - it should be exactly 3 arguments
Use unpacking to access command parts: `a, b, c = [1. 2, 3`